### DISCUSSION TITLE
Typo in testcase 7 slides

### DISCUSSION THREAD

**Discussion by tais958** [NASA-LIS/LISF#1066] [General]
Hi, thanks for the great tutorial of testcase exercises, those really help me familiarize with the logic and procedure. 
Just wanted to report a tiny typo that I found in slide 6 for testcase 7: 

Next run LVT to process the SMAP DA observations to be easily
compared with our model output:
./LVT lvt.config.smapDAobs

• Was the run successful?
o Yes ⇒ Great job!
o No ⇒ look at the error message and **`lvtlog_ol_da.0000`** file   

I think this should be **`lvtlog.smapDAobs.0000`** instead.

Thanks!

**Comment by bmcandr** [comment]
Hi @tais958, thanks for the feedback. Sounds like you are referencing the slides linked on [our website](https://lis.gsfc.nasa.gov/lis-testcases). Those are outdated. A more current Public Testcase Walkthrough resides in our repo under `docs/`: https://github.com/NASA-LIS/LISF/tree/master/docs. The instructions explain how to view the docs in your browser.

We should definitely update the website to point to this new version. Sorry for the confusion.

**Reply by tais958** [reply]
Thanks, @bmcandr. Thanks for pointing me to the updated materials!
